build.gradle

```diff
@@ -301,6 +301,24 @@ task setupIntellijStaticAnalysis {
     dependsOn copyStylelintConfiguration, copyIntelliJProjectPluginsSettings
 }
 
+// IntelliJ project static analysis plugins version sync
+task syncIntelliJCheckStyleVersion {
```

Comment:
Maybe you need to update the developer guide.

I believe the only situation where CheckStyle plugins in IntelliJ is out of sync is we have upgraded to a new version.

Reply:
You are right, I guess I need to.